api/src/test/java/org/openmrs/util/databasechange/Database1_9_7UpgradeIT.java

```diff
@@ -64,6 +66,9 @@
 	
 	private DatabaseUpgradeTestUtil upgradeTestUtil;
 	
+	@ClassRule
+	public static TemporaryFolder tempappdir = TemporaryFolder.builder().assureDeletion().build();
```

Comment:
```suggestion
	public static TemporaryFolder testAppDataDir = TemporaryFolder.builder().assureDeletion().build();
```

the folder field is called differently in this class as you can see from the deleted / remaining lines below

Reply:
@teleivo I revert the latest commit so that I fixed the proposed changes but am scare for unexpected result the PR closed. How can I go about it? I desire working on one one PR.